schema/Organization.re

```diff
@@ -0,0 +1,145 @@
+open Graphql_lwt;
+open GraphqlHelpers;
+
+module Project = {
+  type t = {
+    id: int,
+    name: string,
+    webhook: string,
+  };
+
+  let resolver =
+    Schema.(
+      obj("project", ~doc="Project of an organization", ~fields=_ =>
```

Comment:
```suggestion
      obj("Project", ~doc="Project of an organization", ~fields=_ =>
```
let's enforce type names to be Capitalized

Reply:
I thought in do this, but graphql_lwt uses lower case in some parts so got a bit weird, all mixed, so I thought in keep like this

Reply:
ah yea guys are doing some weird stuff there